server/src/init/middleware.ts

```diff
@@ -0,0 +1,15 @@
+import { Express, json, urlencoded } from "express";
```

Comment:
you could create an index.ts file in middleware and export it with this functionality.